### DISCUSSION TITLE
作者能否共享下阿里云盘新的open接口文档

### DISCUSSION THREAD

**Discussion by yanjunhui** [AlistGo/alist#3724] [General]
目前想去扩展一些功能,  但是看不到官方新的open接口文档, 申请了好几次内测也没通过官方的审核. 能否共享下开发文档, 感激!

**Comment by yanjunhui** [comment]
@Xhofe 不知道能否分享下 感激

**Comment by xhofe** [comment]
https://www.yuque.com/aliyundrive/zpfszx

**Reply by yanjunhui** [reply]
多谢~

**Comment by yanjunhui** [comment]
功能还是很有限啊  我还以为可以完整实现现在的api所有功能呢
哪怕不再像之前的 api 能直接获取分享链接的文件播放权限, 至少能保存别人的分享文件也行啊, 这样也可以曲线救国了, 就目前来看 还是不支持.

**Reply by anwen-anyi** [reply]
哈哈 这个感觉官方不会开放的...

**Reply by yanjunhui** [reply]
是的 现在搞这个新接口就是为了好封锁之前的共享挂载，像以前全走共享挂载了 还咋卖会员

**Reply by anwen-anyi** [reply]
共享挂载本身就是不合规的 别想着了 人家的本来就是正常行为 反而咱们才是不正常的行为.......
